src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -1266,6 +1269,29 @@ ObjectDataHandler::write_ret ObjectDataHandler::write(
     });
 }
 
+ObjectDataHandler::set_alloc_hint_ret
+ObjectDataHandler::set_alloc_hint(
+  context_t ctx,
+  extent_len_t expected_object_size,
+  extent_len_t expected_write_size,
+  uint32_t flags)
+{
+  boost::ignore_unused(expected_write_size);
```

Comment:
What's the goal of expected_write_size ? Or probably add an explicit //TODO comment here?